pkg/probe/exec/exec_test.go

```diff
@@ -110,31 +107,27 @@ func TestExec(t *testing.T) {
 	elevenKilobyte := strings.Repeat("logs-123", 8*128*11) // 8*128*11=11264 = 11KB of text.
 
 	tests := []struct {
-		expectedStatus   probe.Result
-		expectError      bool
-		execProbeTimeout bool
-		input            string
-		output           string
-		err              error
+		expectedStatus probe.Result
+		expectError    bool
+		input          string
+		output         string
+		err            error
 	}{
 		// Ok
-		{probe.Success, false, true, "OK", "OK", nil},
+		{probe.Success, false, "OK", "OK", nil},
 		// Ok
-		{probe.Success, false, true, "OK", "OK", &fakeExitError{true, 0}},
+		{probe.Success, false, "OK", "OK", &fakeExitError{true, 0}},
 		// Ok - truncated output
-		{probe.Success, false, true, elevenKilobyte, tenKilobyte, nil},
+		{probe.Success, false, elevenKilobyte, tenKilobyte, nil},
```

Comment:
Is it intentional to assume the truncated output as `probe.Success`?

Reply:
It's not assuming. The success status is based on the exit code of the probe. The output truncation is orthogonal to that.